tidb-cloud/select-cluster-tier.md

```diff
@@ -29,6 +29,32 @@ For each TiDB Cloud account, you can use one complimentary Developer Tier cluste
 
 The one-year free trial begins the day the first Developer Tier cluster is created.
 
+### User name prefix
+
+<!--Important: Do not update the section name "User name prefix" because this section is referenced by TiDB backend error messages.-->
+
+For each Developer Tier cluster, TiDB Cloud generates a unique prefix to distinguish it from other clusters.
+
+Whenever you use or create a database user name, you must include the prefix in the user name. For example, assume that the prefix of your cluster is `3pTAoNNegb47Uc8`:
+
+- To connect to your cluster
+
+    ```shell
+    mysql --connect-timeout 15 -u '3pTAoNNegb47Uc8.root' -h <host> -P 4000 -D test -p
+    ```
+
+- To create a database user
```

Comment:
```suggestion
- To create a database user:
```

Reply:
the database user is not `jefferey`, it's `3pTAoNNegb47Uc8.jeffrey`